docsteady/utils.py

```diff
@@ -115,11 +115,18 @@ def owner_for_id(owner_id):
     if owner_id not in Config.CACHED_USERS:
         resp = requests.get(Config.USER_URL.format(username=owner_id),
                             auth=Config.AUTH)
-        resp.raise_for_status()
-        user_resp = resp.json()
```

Comment:
I think all you need to do here is replace these two lines with:
```
if resp.status_code == 404:
    user_resp = {'displayName': owner_id}
else:
    resp.raise_for_status()
    user_resp = resp.json()
```
instead of your more complex change.